src/bin/thermometer.rs

```diff
@@ -221,19 +279,21 @@ async fn main(spawner: Spawner) {
     let scl = peripherals.GPIO8;
 
     let i2c = mk_static!(
-        I2c<'static, Blocking, AnyI2c>,
+        I2c<'static, Blocking>,
         I2c::new(
             peripherals.I2C0,
-            Config {
-                frequency: 100.kHz(),
-                ..Default::default()
-            }
+            Config::default().with_frequency(100.kHz())
         )
+        .unwrap()
```

Comment:
Not sure how likely it is that something goes wrong here, but should we turn this into an `expect` maybe for some additional context?

Reply:
It should never happen in our specific case, in theory.